app/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import "../styles/globals.css";
+
+type Props = {
+  children: React.ReactNode;
+};
+const RootLayout = ({ children }: Props) => {
```

Comment:
저는 개행도 읽기 좋은 코드를 작성하는데 중요하다고 생각해서 이렇게 챙겨주면 좋을 것 같아요~
```suggestion
};

const RootLayout = ({ children }: Props) => {
```